src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +238,57 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| Combined PD/T2 map                           | PDT2map   | In arbitrary units (arbitrary). Combined PD/T2 maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                                                                                                                                                                               |
+| Quantitative susceptibility map (QSM)        | Chimap    | In parts per million (ppm). QSM allows for determining the underlying magnetic susceptibility of tissue (Chi) ([Wang & Liu, 2014](https://onlinelibrary.wiley.com/doi/10.1002/mrm.25358)). Chi maps are REQUIRED to use this suffix regardless of the method used to generate them.                                                                                                                                                                                                                               |
+| RF transmit field map                        | TB1map    | In arbitrary units (arbitrary). Radio frequency (RF) transmit (B1+) field maps are REQUIRED to use this suffix regardless of the method used to generate them. TB1map intensity values are RECOMMENDED to be represented as percent multiplicative factors such that FlipAngle<sub>effective</sub> = B1+<sub>intensity</sub>*FlipAngle<sub>nominal</sub> .                                                                                                                                                                                                                                                                                                                                                                    |
+| RF receive sensitivity map                   | RB1map    | In arbitrary units (arbitrary). Radio frequency (RF) receive (B1-) sensitivity maps are REQUIRED to use this suffix regardless of the method used to generate them. RB1map intensity values are RECOMMENDED to be represented as percent multiplicative factors such that Amplitude<sub>effective</sub> = B1-<sub>intensity</sub>*Amplitude<sub>ideal</sub>.                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Just to clarify, are these supposed to go in `anat`? They seem like they would fall under `fmap`.